spyder/widgets/ipythonconsole/shell.py

```diff
@@ -92,7 +94,11 @@ def is_spyder_kernel(self):
 
     def set_cwd(self, dirname):
         """Set shell current working directory."""
-        code = u"get_ipython().kernel.set_cwd(r'{}')".format(dirname)
+        # Replace single for double backslashes on Windows
+        if os.name == 'nt':
+            dirname = dirname.replace(u"\\", u"\\\\")
```

Comment:
Ok, sure. Its help doesn't mention that it converts `\` to `\\`, but please open a PR for this. If tests pass then I'm for it.